doc/index.md

```diff
@@ -0,0 +1,25 @@
+---
+# https://vitepress.dev/reference/default-theme-home-page
+layout: home
+
+hero:
+  name: "wire"
+  text: "a tool to deploy nixos systems"
+  # tagline: My great project tagline
+  actions:
+    - theme: brand
+      text: Read Guide
+      link: /guide/wire
+    - theme: alt
+      text: Reference
+      link: /reference/cli
+
+features:
+  - title: Parallelism
+    details: Build and deploy many nodes at once
+  - title: Secrets & Keys
+    details: Fast keys that never enter your store
+  - title: Node Tagging & CI Friendly
```

Comment:
may be appropriate to separate these two in the future maybe, unsure!

Reply:
In the future all of these will have fleshed out pages as well, i just wanna scaffold the website those can come later